### DISCUSSION TITLE
Auto generate tables on application run?

### DISCUSSION THREAD

**Discussion by ghost** [prisma/prisma#8002] [Q&A] [answered]
Hi! I have a project that consists of multiple JS files. My program runs the code from each JS file when opened. Is there a way to auto-generate the database tables (without migrate) when I open the program, and when the program reads the JS file, where most of my Prisma stuff is?

**Comment by ryands17** [comment, marked as answer]
@cuBeUrg 👋 

Could you provide an example as I didn't quite get your use case?

**Reply by ghost** [reply]
So, just to simplify the question, is there a function that auto-generates the tables in the database, without using Migrate?

**Reply by ryands17** [reply]
Yes, have a look at this: https://www.prisma.io/docs/concepts/components/prisma-migrate/db-push

**Reply by ghost** [reply]
Ok, thanks! This is what I've been looking for.
